### DISCUSSION TITLE
Skip empty sections in chunkedhtml navigation and TOC?

### DISCUSSION THREAD

**Discussion by romanticopera** [jgm/pandoc#9889] [Q&A]
Is there any way to skip empty chapter headings when generating the TOC and navigation links of a chunkedhtml project? 

For example, in the demo (https://pandoc.org/chunkedhtml-demo/), "3 Options" is an empty page.

I'm trying to create a project where the equivalent of "3 Options" is plain-text (not a link) and section 2.5's "Next" link goes directly to 3.1 rather than the empty "3 Options."

Is there any way to do this?

**Comment by jgm** [comment]
I agree that the blank page is a bit odd.  I think it would be nicer if it worked like GNU texinfo:
https://www.gnu.org/software/emacs/manual/html_node/emacs/Basic.html
where you get a mini-table of contents for the section, even if there's no text.
